client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -68,6 +67,21 @@ def test_get_status(self, client):
         assert "date_created" in status
         assert "id" in status
 
+    def test_repr(self, client, experiment_run, model_for_deployment):
+        experiment_run.log_model(model_for_deployment['model'], custom_modules=[])
+        experiment_run.log_requirements(['scikit-learn'])
+
+        path = verta._internal_utils._utils.generate_default_name()
+        endpoint = client.set_endpoint(path)
+        endpoint.update(experiment_run, DirectUpdateStrategy())
+
+        str_repr = str(endpoint)
```

Comment:
Pretty sure this makes no difference at all, but at least for consistency:
```suggestion
        str_repr = repr(endpoint)
```

Reply:
fixed.